plugin.py

```diff
@@ -1,66 +1,30 @@
 import shutil
 import os
 import sublime
-import threading
-import subprocess
 
 from LSP.plugin.core.handlers import LanguageHandler
-from LSP.plugin.core.settings import ClientConfig, LanguageConfig, read_client_config
+from LSP.plugin.core.settings import ClientConfig
+from LSP.plugin.core.settings import read_client_config
```

Comment:
I think I prefer two above on the same line :)

Reply:
not sure about this as I cloned these lines from https://github.com/sublimelsp/LSP-eslint/blob/master/plugin.py#L8-L9

Reply:
Yes, it needs cleanup too, but here let's just do it already:
```py
from LSP.plugin.core.settings import ClientConfig, read_client_config
```

Reply:
ok!